data/json/items/comestibles/vegetable.json

```diff
@@ -0,0 +1,312 @@
+    "fun": -1
+  },
+  {
+    "id": "potato_raw",
+    "copy-from": "vegetable_root",
+    "type": "COMESTIBLE",
+    "name": "raw potato",
+    "name_plural": "raw potatoes",
+    "description": "Mildly toxic and not very tasty raw.  When cooked, it is delicious.",
+    "weight": 210,
+    "volume": 1,
+    "price": 90,
+    "nutrition": 8,
+    "healthy": -2,
+    "fun": -3
+  },
+  {
+    "id": "pumpkin",
+    "copy-from": "vegetable",
+    "type": "COMESTIBLE",
+    "name": "pumpkin",
+    "description": "A large vegetable, about the size of your head.  Not very tasty raw, but is great for cooking.",
+    "weight": 600,
+    "volume": 12,
+    "price": 750,
+    "charges": 8,
+    "spoils_in": 240,
+    "nutrition": 22,
+    "quench": 5,
+    "healthy": 2,
+    "fun": -1
+  },
+  {
+    "id" : "raw_beans",
+    "copy-from": "vegetable",
+    "type" : "COMESTIBLE",
+    "name" : "raw beans",
+    "name_plural" : "raw beans",
+    "description" : "Raw, uncooked beans.  They are mildly toxic in this form, but you could cook them to make them tasty.  Alternatively, you could plant them.",
+    "weight" : 210,
+    "volume" : 1,
+    "price" : 90,
+    "nutrition" : 14,
+    "healthy" : -2,
+    "fun" : -1,
+    "seed_data": {
+        "plant_name": "beans",
+        "fruit": "raw_beans",
+        "byproducts": [ "withered" ],
+        "grow" : 91
+    }
+  },
+  {
+    "id": "raw_dandelion",
+    "copy-from": "vegetable_leafy",
+    "type": "COMESTIBLE",
+    "name": "handful of dandelions",
+    "name_plural": "handfuls of dandelions",
+    "description": "A collection of freshly picked yellow dandelions.  In their current raw state they are quite bitter.",
+    "weight": 96,
+    "volume": 1,
+    "price": 60,
+    "charges": 2,
+    "stack_size": 4,
+    "nutrition": 4,
+    "healthy": 1,
+    "fun": -12
+  },
+  {
+    "id": "rhubarb",
+    "copy-from": "vegetable",
+    "type": "COMESTIBLE",
+    "name": "rhubarb",
+    "description": "Sour stems of the rhubarb plant, often used in baking pies.",
+    "weight": 300,
+    "volume": 1,
+    "price": 410,
+    "charges": 4,
+    "spoils_in": 160,
+    "nutrition": 8,
+    "quench": 3,
+    "healthy": 3
+  },
+  {
+    "id": "sugar_beet",
+    "copy-from": "vegetable",
+    "type": "COMESTIBLE",
+    "name": "sugar beet",
+    "description": "This fleshy root is ripe and flowing with sugars; just takes some processing to extract them.",
+    "weight": 310,
+    "volume": 4,
+    "price": 50,
+    "charges": 4,
+    "spoils_in": 2280,
+    "nutrition": 5,
+    "fun": -10,
+    "stim": 1
+  },
+  {
+    "id": "veggy_wild",
+    "copy-from": "vegetable_leafy",
+    "type": "COMESTIBLE",
+    "name": "wild vegetables",
+    "name_plural": "wild vegetables",
+    "description": "An assortment of edible-looking wild plants.  Most are quite bitter-tasting.",
+    "weight": 238,
+    "volume": 1,
+    "price": 60,
+    "spoils_in": 80,
+    "nutrition": 20,
+    "healthy": 1,
+    "fun": -10
+  },
```

Comment:
That was lost a little while back. IRL the risks are exceptionally low (to nil). I'd like to rework a better parasitism system rather than bolt the flawed one back on.